frontend/src/App.js

```diff
@@ -1,96 +1,99 @@
-import React, {Component} from "react";
-import {
-  HashRouter as Router,
-  Route,
-  Switch,
-  Redirect
-} from "react-router-dom";
-import CssBaseline from "@material-ui/core/CssBaseline";
-import { MuiThemeProvider } from "@material-ui/core/styles";
-import { connect } from "react-redux";
-
-import Home from "./containers/Home";
-import Users from "./containers/Users";
-import LoginPage from "./containers/Signin";
-
-import MainLayout from "./layouts/MainLayout";
-import EmptyLayout from "./layouts/EmptyLayout";
-
-const NotFound = () => {
-  return <div>NotFound</div>;
-};
-
-
-const DashboardRoute = ({ component: Component, ...rest }) => {
-  return (
-    <Route
-      {...rest}
-      render={matchProps => (
-        <MainLayout>
-          <Component {...matchProps} />
-        </MainLayout>
-      )}
-    />
-  );
-};
-
-const EmptyRoute = ({ component: Component, ...rest }) => {
-  return (
-    <Route
-      {...rest}
-      render={matchProps => (
-        <EmptyLayout>
-          <Component {...matchProps} />
-        </EmptyLayout>
-      )}
-    />
-  );
-};
-
-class App extends Component {
-  state = {
-    auth: false
-  };
-
-  render() {
-    const { auth } = this.props;
-
-    return (
-        <MuiThemeProvider>
-          <CssBaseline />
-          <div style={{ height: "100vh" }}>
-            <Router>
-              {auth.authenticate ? (
-                <Switch>
-                  <DashboardRoute path="/dashboard" component={Home} />
-                  <DashboardRoute path="/users" component={Users} />
-                  <Route path="/signin" render={() => <Redirect to="/" />} />
-                  <DashboardRoute exact path="/" component={Home} />
-                  <EmptyRoute component={NotFound} />
-                </Switch>
-              ) : (
-                <Switch>
-                  <EmptyRoute path="/signin" component={LoginPage} />
-                  <Redirect to="/signin" />
-                </Switch>
-              )}
-            </Router>
-          </div>
-        </MuiThemeProvider>
-    );
-  }
-}
-
-App.propTypes = {};
-
-const mapStateToProps = state => {
-  return {
-    settings: state.settings,
-    auth: state.auth,
-  };
-};
-
-export default connect(
-  mapStateToProps,
-  null
-)(App);
+import React, {Component} from "react";

```

Comment:
It looks like most of the changes in this file have to do with the end of line character. I deduce this because most of the lines look the same but git is detecting a change. This makes finding and reviewing the real changes difficult, and conflicts are more likely because more likes have been (needlessly?) modified.

Let's get those changes out. You might need to revert the commit that made this change. And then redo the real changes, while making sure that git (and maybe your IDE) is configure to handle end-of-line in a sane manner. Here is a good article on git and EOL https://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/ . We might want to add a .gitattributes file for this project in a different PR.